app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,36 @@
+<header class="container border-bottom">
+  <nav class="navbar navbar-expand-lg navbar-light justify-content-between">
+    <a class="navbar-brand" href="#">Recipe</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
```

Comment:
The navbar toggle menu is not functioning well since nothing happens when you click on it. Kindly ensure that it works to provide a good user experience for users.